src/server/config/environment/development.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   apiUrl: 'http://localhost:3001',
   webUrl: 'http://localhost:3002',
+  GA_TRACKING_ID: 'GA_TRACKING_ID',
```

Comment:
@Mar1nka please rename this variable to match our styles and prevent issues with linter in the future (`GA_TRACKING_ID` => `gaTrackingId`)